src/components/EditPostModal/index.tsx

```diff
@@ -111,6 +116,7 @@ const EditPostModal = ({
   };
 
   const handleEditPost = async () => {
+    const mentionees = mentionUsers.map((user) => user.id);
     if (displayImages.length > 0) {
       const fileIdArr: (string | undefined)[] = displayImages.map(
```

Comment:
Should be this?

```suggestion
      const fileIdArr: string[] | undefined = displayImages.map(
```

Reply:
I checked current code structure and we need undefined array. 
These undefined will be handled in editPost function.

Reply:
ok